client/components/PostsList.jsx

```diff
@@ -7,7 +7,7 @@ const PostsList = ({ setPostFormContent, postContent }) => {
 
 
         postContent.map((post) => {
-            return <Post key={post._id} setPostFormContent={setPostFormContent} userId={post.userId} postId={post._id} profileName={post.profileName} postContent={post.postContent} upVote={'up vote'} downVote={'down vote'} comment={`${post.commentsCount} comments`} timeStamp={new Date(post.createdAt).toLocaleString('en-GB', { hour: 'numeric', minute: 'numeric', second: 'numeric', year: 'numeric', month: 'numeric', day: 'numeric' })} />
+            return <Post key={post._id} setPostFormContent={setPostFormContent} userId={post.userId} postId={post._id} profileName={post.profileName} postContent={post.postContent} comment={`${post.commentsCount} comments`} timeStamp={new Date(post.createdAt).toLocaleString('en-GB', { hour: 'numeric', minute: 'numeric', second: 'numeric', year: 'numeric', month: 'numeric', day: 'numeric' })} />
```

Comment:
There's way too much props here,
can u make it better indentation ?
like this:
```
return <Post 
key={key}
bla={bla}
tick={tack}
>
```